### PR TITLE
Update to API 2019-07-01 (#48)

### PR DESCRIPTION
**Pull request by leonardbf**
NFSv4 is now NFSv4.1
Snapshot date name change.
Creation-token rename to file-path
Internal ticket ANF-245 for R5 RP

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/setup.py [hunk 1]
```diff
@@ -104,7 +104,7 @@
     'azure-mgmt-monitor~=0.5.2',
     'azure-mgmt-msi~=0.2',
     'azure-mgmt-network~=4.0',
-    'azure-mgmt-netapp~=0.5.0',
+    'azure-mgmt-netapp~=0.6.0',
```
Comment (Juliehzl) [inline src/azure-cli/setup.py]: You also need to upgrade the packages in requirementsxxx.txt file.

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -72,6 +72,13 @@ Release History
 
 * monitor metrics alert create: Fix #9901. Support special character `:` in `--condition` argument.
 
+**NetAppFiles**
+
+* Update for API version 2019-07-01 :
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: Can you re-edit your release note  in the following words:
* Upgrade azure-mgmt-netapp to 0.6.0 to use API version 2019-07-01, with the new API version:

    * Volume creation --protocol-types accepts now "NFSv4.1" not "NFSv4"
    * Volume export policy property now named 'nfsv41' not 'nfsv4'
    * Snapshot creation date now named just 'created'

Feel free to re-edit as what you  expected, just my suggestion.

**Comment by yonzhan** [comment]
@leonardbf please take a look at zunli's comments and improve your code.

File: src/azure-cli/setup.py (hunk 1 above)
Comment (leonardbf) [reply src/azure-cli/setup.py]: ok. I've updated what I hope are the correct files.

File: src/azure-cli/HISTORY.rst (hunk 2 above)
Comment (leonardbf) [reply src/azure-cli/HISTORY.rst]: ok. Updated to use similar wording.

**Comment by Juliehzl** [comment]
Hi @leonardbf, due to the update in dev branch, you need to resolve many conflicts to make CI pass. Could you resolve conflicts and make CI pass?

**Pull request closed without merging**